user_journey/views/event_type.view.lkml

```diff
@@ -0,0 +1,143 @@
+view: event_type {
+  derived_table: {
+    sql:
+    SELECT
```

Comment:
This is such a horrible SQL statement because it cannot use CTEs. It's basically doing this:

SELECT
   aggregate over all event_types
   FROM
     SELECT
       ping_type,
       aggregate over all event_properties, '.' if they were not selected to be filtered
       FROM
         all_event_properties
       LEFT JOIN
         selected_event_properties_for_filtering

Reply:
Also, the templated filters prevent me from pulling it into a UDF, or I'd go that route.